sound/soc/intel/boards/cnl_rt700.c

```diff
@@ -250,6 +248,13 @@ struct snd_soc_dai_link cnl_rt700_msic_dailink[] = {
 #endif
 };
 
+static struct snd_soc_codec_conf rt1308_codec_conf[] = {
+	{
+		.dev_name = "sdw:2:25d:1308:0:0",
+		.name_prefix = "rt1308",
```

Comment:
the prefix is not going to scale if you have 2 rt1308, so need to have rt1308-1 or something